dask/rewrite.py

```diff
@@ -133,8 +133,11 @@ class RewriteRule(object):
     ----------
     lhs : task
         The left-hand-side of the rewrite rule.
-    rhs : task
-        The right-hand-side of the rewrite rule.
+    rhs : task or function
+        The right-hand-side of the rewrite rule. If it's a task, variables in
+        `rhs` will be replaced by terms in the subject that match the variables
+        in `lhs`. If it's a function, the function will be called with a dict
+        of such matches.
```

Comment:
Is there an easy example one could add to the RewriteRule docstring showing how this works?